src/main/resources/modules/uti/telemed_path.json

```
@@ -0,0 +1,535 @@
+          }
+        }
+      ]
+    },
+    "Lab Encounter": {
+      "type": "Encounter",
+      "encounter_class": "ambulatory",
+      "reason": "uti",
+      "telemedicine_possibility": "none",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 185347001,
+          "display": "Encounter for problem (procedure)",
+          "value_set": ""
+        }
+      ],
+      "direct_transition": "Labs Sub",
+      "name": "Lab Encounter"
+    },
+    "Labs Sub": {
+      "type": "CallSubmodule",
+      "submodule": "uti/labs",
+      "direct_transition": "End Labs Encounter",
+      "name": "Labs Sub"
+    },
+    "End Labs Encounter": {
+      "type": "EncounterEnd",
+      "direct_transition": "Result Delay",
+      "name": "End Labs Encounter"
+    },
+    "Result Delay": {
+      "type": "Delay",
+      "distribution": {
+        "kind": "UNIFORM",
+        "parameters": {
+          "high": 48,
+          "low": 24
+        }
+      },
+      "unit": "hours",
+      "direct_transition": "Lab Result Encounter",
+      "name": "Result Delay"
+    },
+    "Lab Result Encounter": {
+      "type": "Encounter",
+      "encounter_class": "virtual",
+      "reason": "uti",
+      "telemedicine_possibility": "always",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 185316007,
+          "display": "Indirect encounter (procedure)",
+          "value_set": ""
+        }
+      ],
+      "direct_transition": "Follow Up",
+      "name": "Lab Result Encounter"
+    },
+    "Follow Up": {
+      "type": "CallSubmodule",
+      "submodule": "uti/lab_follow_up",
+      "direct_transition": "End Results Encounter",
+      "name": "Follow Up"
+    },
+    "End Results Encounter": {
+      "type": "EncounterEnd",
+      "direct_transition": "Terminal",
+      "name": "End Results Encounter"
+    },
+    "Patient Choice": {
+      "type": "Simple",
+      "name": "Patient Choice",
+      "complex_transition": [
+        {
+          "condition": {
+            "condition_type": "And",
+            "conditions": [
+              {
+                "condition_type": "Attribute",
+                "attribute": "pregnant",
+                "operator": "==",
+                "value": true
+              },
+              {
+                "condition_type": "Active Condition",
+                "codes": [
+                  {
+                    "system": "SNOMED-CT",
+                    "code": 45816000,
+                    "display": "Pyelonephritis"
+                  }
+                ]
+              }
+            ]
+          },
+          "distributions": [],
+          "transition": "Emergency Follow Up"
+        },
+        {
+          "distributions": [
+            {
+              "transition": "Emergency Follow Up",
+              "distribution": 0.1
+            },
+            {
+              "transition": "Ambulatory Encounter",
+              "distribution": 1
```

```suggestion
              "distribution": 0.9
```

Technically this doesn't matter since the engine logic will correct this internally, but it's nice to have the right number show